src/pysonar_scanner/configuration/cli.py

```diff
@@ -451,6 +451,30 @@ def __parse_cli_args(cls) -> tuple[argparse.Namespace, list[str]]:
             type=bool,
             help="Equivalent to -Dsonar.python.xunit.skipDetails",
         )
+        parser.add_argument(
+            "--sonar-python-mypy-report-paths",
```

Comment:
Could it make sense to have shorter aliases as well, like `mypy-report-paths`, etc...?
I think it makes sense to keep the original property name converted from camel case, but it would probably be more practical to have these as well.